app/sgqlc_schema.py

```diff
@@ -27,30 +51,14 @@ class CaseFilter(Node):
     age_category = String
 
 
-# QUERY USE ONLY
-class SubCaseNode(Node):
-    species = String
-
-
-class SubManifestNode(Node):
-    filename = String
-    is_derived_from = list_of(String)
-    is_source_of = list_of(String)
-
-
-class SubDatasetDescriptionNode(Node):
-    title = String
-    subtitle = String
-    study_organ_system = list_of(String)
-    number_of_subjects = Int
-    number_of_samples = Int
-    keywords = list_of(String)
-
-
+# QUERY/PAGINATION USE ONLY
 class ExperimentNode(Node):
     submitter_id = String
     dataset_descriptions = list_of(SubDatasetDescriptionNode)
-    manifests = list_of(SubManifestNode)
+    manifests1 = list_of(SubManifestNode)
+    manifests2 = list_of(SubManifestNode)
+    manifests3 = list_of(SubManifestNode)
+    manifests4 = list_of(SubManifestNode)
```

Comment:
Are these used anywhere?

Reply:
You can find under the update_manifests_information function in sgqlc.py file. These are used to fetch different types of manifest file.